terraform/datadog/cve.tf

```diff
@@ -0,0 +1,19 @@
+resource "datadog_monitor" "cve-reporter" {
+  count = "${var.enable_cve_monitor}"
+
+  name    = "New CVE Reported"
```

Comment:
I think it's worth prefixing this name with the `deploy_env` for consistency. I realise they we'll only typically only ever have one instance of this deployed, but while developing etc, there may be more than one, and adding the env would make it easier to see where they came from.